Lannister/App/Controllers/ETHCreateHoldingController.swift

```diff
@@ -0,0 +1,171 @@
+//
+//  ETHCreateHoldingController.swift
+//  Lannister
+//
+//  Created by André Sousa on 08/07/2019.
+//  Copyright © 2019 André Sousa. All rights reserved.
+//
+
+import UIKit
+import QRCodeReader
+
+protocol ETHDelegate {
+    func importedAddress(address: String)
+    func insertedValue(value: Double)
+}
+
+class ETHCreateHoldingController: UIViewController {
+    
+    @IBOutlet weak var valueTextField   : UITextField!
+    @IBOutlet weak var tableView        : UITableView!
+    var toolBar                         = UIToolbar()
+    var delegate                        : ETHDelegate!
+    var holdingValue                    : Double!
+
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        navigationItem.title = "ETH"
+
+        navigationController!.navigationBar.titleTextAttributes =
+            [NSAttributedString.Key.font: UIFont(name: "AvenirNext-Medium", size: 18)!,
+             NSAttributedString.Key.foregroundColor : UIColor(red: 118/255, green: 134/255, blue: 162/255, alpha: 1)]
```

Comment:
Can we refactor these colors since they're the same? Are we setting the colors somewhere globally?

Reply:
Not yet! But maybe we can handle it as a separate issue/PR.